source/content/guides/composer-convert.md

```diff
@@ -190,49 +247,49 @@ composer prepare-for-pantheon
 composer install --no-dev
 ```
 
-This should modify the `.gitignore` file and cleanup any errant `.git` directories in the codebase, to prepare your new code for direct deployment to Pantheon.
+This modifies the `.gitignore` file and cleans up any errant `.git` directories in the codebase to prepare your new code for direct deployment to Pantheon.
 
-## Commit
+## Deploy
 
-Commit your work to the git repo. From the `$site` directory, run the following:
+You've now committed the code to the local branch. If your site has Multidev, you can deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composerify -- wd-show` to inspect the watchdog logs, or follow the directions in our documentation on [log collection](/logs).
```

Comment:
This paragraph with `drush wd-show` could be moved up to the "Set up a Multidev" section above. The rest of this ("Deploy" and "Deploy to a Multidev") is redundant with what has come before.